TestProject/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿using MarsQA_Nunit.Models;
+using MarsQA_Nunit.Pages;
+using MarsQA_Nunit.Utilities;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace MarsQA_Nunit.Tests
+{
+    [TestFixture]
+    public class CertificationTests : CommonDriver
+    {
+        Login login = new Login();
+        Certifications certification = new Certifications();
+        private readonly By certificationTabLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[1]/a[4]");
+        IWebElement certificationTab;
+       
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            var testData = GetTestData();
+            login.LoginActions(driver, testData.LoginData);
+            CertificationTab();
+            
+        }
+        public void CertificationTab()
+        {
+            //Click on Certification Tab
+            IWebElement certificationTab = driver.FindElement(certificationTabLocator);
+            certificationTab.Click();
+            DataCleanUp();
```

Comment:
i would move this in a hook that runs before any tests

Reply:
note you need to clean call data before any test runs and then clean data that the test generates at the end of each test! (do not clean all the data but only the one that the test is generating).

Reply:
in your teardown you can do (do not clean all the data but only the one that the test is generating).